shared/people/invite-friends/tab-bar-button.tsx

```diff
@@ -23,11 +23,12 @@ const InviteFriends = () => {
       </Kb.Text>
     </Kb.Box2>
   )
+  const firstTooltipLine = `${num.toLocaleString()} friends invited in the last 24 hours.`
   return Styles.isMobile ? (
     <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.mobileContainer}>
       <Kb.Box style={Styles.globalStyles.flexOne} />
       {inviteButton}
-      {inviteCounter}
+      <Kb.WithTooltip tooltip={firstTooltipLine}>{inviteCounter}</Kb.WithTooltip>
```

Comment:
Something is off with this layout on mobile now :/ 
![image](https://user-images.githubusercontent.com/11968340/77085934-69139e80-69d7-11ea-875c-bad1e59e089a.png)

Reply:
oops thanks for thorough testing
fixed now, boxes within boxes within boxes